README.md

```diff
@@ -29,17 +29,20 @@ If you are using Maven to manage your project's dependencies, you can run `mvn i
 If you aren't using Maven, you can add the `jar` in the normal way.  This `jar` can be found in
 the `target` directory created when you ran `mvn package`.
 
-Note that this will *not* run the integration tests, since these require access to a Conjur instance.  To run the
-integration tests, you will need to define the following environment variables for the `mvn package` command 
-(and remove the `skipTests` property):
+Note that this *did not* run the integration tests, since these require access to a Conjur instance. You can run the 
+integration tests with `mvn package` once you finished with the configuration.
+
+### Configuration
+
+The simplest way to configure the Conjur API is to define the following environment variables:
 
 ```bash
 CONJUR_ACCOUNT=accountName
 CONJUR_CREDENTIALS=username:apiKey
 CONJUR_APPLIANCE_URL=http://conjur
```

Comment:
Would be good to put more clarity around the values expected in these variables. Otherwise I fear some folk will copy and paste them as-is, which obviously won't work. Also, are they the same variable names as other libraries use?

Reply:
I'll add some. I also want to change CONJUR_CREDENTIALS to 2 different variable - conjur_authn_login & conjur_authn_api_key - as this is the more common way throughout conjur. I'll do this in another PR as it changes a lot of code